contracts/cw-core/src/state.rs

```diff
@@ -49,8 +67,15 @@ pub const PAUSED: Item<Expiration> = Item::new("paused");
 
 /// The voting module associated with this contract.
 pub const VOTING_MODULE: Item<Addr> = Item::new("voting_module");
-/// The proposal modules assocaited with this contract.
-pub const PROPOSAL_MODULES: Map<Addr, Empty> = Map::new("proposal_modules");
+
+/// The proposal modules associated with this contract.
+pub const PROPOSAL_MODULES: Map<Addr, ProposalModule> = Map::new("proposal_modules_v1");
```

Comment:
```suggestion
pub const PROPOSAL_MODULES: Map<Addr, ProposalModule> = Map::new("proposal_modules_v2");
```

Reply:
Might be nice to add a comment here about why we don't use the key `proposal_modules` for future viewers.